dataeng/jobs/analytics/SnowflakeSchemaBuilder.groovy

```diff
@@ -44,14 +44,8 @@ class SnowflakeSchemaBuilder {
             publishers common_publishers(allVars)
             steps {
 
-                virtualenv {
-                    pythonName('PYTHON_3.7')
```

Comment:
Installed python3.8 version from shipping panda plugin is not working properly.
I saw some places where we are using the following syntax to provision python3.8 virtual env